platforms/osx/src/main.mm

```diff
@@ -13,11 +13,9 @@
 
 std::string sceneFile = "scene.yaml";
 
-std::string markerStyling = "{ style: 'points', interactive: true, color: 'white', size: [25px, 25px], order: 100, collide: false }";
+std::string markerStyling = "touch.point.icons";
```

Comment:
Sometimes its better to be consistent that to be correct ;) Or in other words I'm also in favor of explicitly including all path elements (as with scene updates) to avoid any potential confusion for the developer.